swagger2.0/oic.wk.rd.swagger.json

```diff
@@ -1,598 +1,476 @@
+    }
+  },
+  "definitions": {
+    "rdSelection" : {
+      "properties": {
+        "rt" : {
+          "description": "Resource Type of the Resource",
+          "items": {
+            "enum": ["oic.wk.rd"],
+            "type": "string"
+          },
+          "minItems": 1,
+          "maxItems": 1,
+          "uniqueItems": true,
+          "readOnly": true,
+          "type": "array"
+        },
+
+        "n" : {
+          "$ref": "https://openconnectivityfoundation.github.io/core/schemas/oic.common.properties.core-schema.json#/definitions/n"
+        },
+
+        "sel" : {
+          "description": "A bias factor calculated by the Resource directory",
+          "maximum": 100,
+          "minimum": 0,
+          "readOnly": true,
+          "type": "integer"
+        },
+
+        "id" : {
+          "$ref": "https://openconnectivityfoundation.github.io/core/schemas/oic.common.properties.core-schema.json#/definitions/id"
+        },
+
+        "if" : {
+          "description": "The interface set supported by this resource",
+          "items": {
+            "enum": [
+              "oic.if.baseline",
+              "oic.if.ll",
+              "oic.if.b",
+              "oic.if.lb",
+              "oic.if.rw",
+              "oic.if.r",
+              "oic.if.a",
+              "oic.if.s"
+            ],
+            "type": "string"
+          },
+          "minItems": 1,
+          "readOnly": true,
+          "type": "array"
+        }
+      },
+      "type" : "object",
+      "required": ["sel"]
+    },
+
+    "rdPublish" : {
+      "properties": {
+        "rt" : {
+          "description": "Resource Type of the Resource",
+          "items": {
+            "enum": ["oic.wk.rd"],
+            "enum": "oic.rd.publish"
+          },
+          "minItems": 1,
+          "maxItems": 1,
+          "uniqueItems": true,
+          "readOnly": true,
+          "type": "array"
+        },
+
+        "links" : {
+          "description": "A set of simple or individual OIC Links.",
+          "items": {
+            "properties": {
+              "anchor": {
+                "$ref": "https://openconnectivityfoundation.github.io/core/schemas/oic.links.properties.core-schema.json#/definitions/anchor"
+              },
+              "di": {
+                "$ref": "https://openconnectivityfoundation.github.io/core/schemas/oic.links.properties.core-schema.json#/definitions/di"
+              },
+              "eps": {
+                "$ref": "https://openconnectivityfoundation.github.io/core/schemas/oic.links.properties.core-schema.json#/definitions/eps"
+              },
+              "href": {
+                "$ref": "https://openconnectivityfoundation.github.io/core/schemas/oic.links.properties.core-schema.json#/definitions/href"
+              },
+              "if": {
+                "description": "The interface set supported by this resource",
+                "items": {
+                  "enum": [

```

Comment:
Again, why the full list of OCF Interfaces?

Reply:
See earlier comment re: RD needing further rework as noted on the call.